src/ILCompiler.Compiler/src/Compiler/ModuleClassConstructors.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+using Internal.TypeSystem;
+
+using AssemblyName = System.Reflection.AssemblyName;
+using Debug = System.Diagnostics.Debug;
+
+namespace ILCompiler
+{
+    /// <summary>
+    /// Encapsulates a list of class constructors that must be run in a prescribed order during start-up
+    /// </summary>
+    public sealed class ModuleClassConstructors
```

Comment:
(Definitely just a nit): Simply calling the methods `InitializeClass` or something like that would also help a bit: the `ILT_` prefix contributes to making this look unnatural. `ILT_` stands for ILTransform, which is not what we're doing.